extensions/stats/config.proto

```diff
@@ -18,20 +18,22 @@ syntax = "proto3";
 package stats;
 
 message PluginConfig {
-  // Indicates whether the module should act as inbound or outbound reporter.
-  enum Direction {
-    INBOUND = 0;
-    OUTBOUND = 1;
-  }
-  Direction direction = 1;
-
   // The following settings should be rarely used.
-
-  // it will enable debug for this filter.
-  bool debug = 2;
+  // Enable debug for this filter.
+  bool debug = 1;
 
   // maximum size of the peer metadata cache.
   // A long lived proxy that connects with many transient peers can build up a
   // large cache.
-  int32 max_peer_cache_size = 3;
+  int32 max_peer_cache_size = 2;
+
+  // prefix to add to stats emitted by the plugin.
+  string stat_prefix = 3;  // default: "istio_"
+
+  // Stats api squashes dimensions in a single string.
+  // The squashed string is parsed at prometheus scrape time to recover
+  // dimensions. The following 2 fields set the field and value separators {key:
+  // value} -->  key{value_separator}value{field_separator}
+  string field_separator = 4;  // default: ";;"
```

Comment:
We had talked about using ;.

Reply:
semicolon comma

Reply:
The actual default is `;.;` will fix this.